build/yamls/flow-aggregator.yml

```diff
@@ -65,7 +73,7 @@ spec:
         command:
         - flow-aggregator
         image: antrea/flow-aggregator
-        imagePullPolicy: IfNotPresent
+        imagePullPolicy: Always
```

Comment:
should we just stick to antrea's model of putting onus on the user to update the image? It could be inefficient to keep pulling the same image even though there is no change in it.

Reply:
Thanks, addressed.